src/App.js

```diff
@@ -130,26 +131,30 @@ class Bloc extends Component {
 
 const ColorButts = ({ background, onChangeBackground }) => {
   return (
```

Comment:
You dont have logic in this component. So you can use implicit return
```
const ColorButts = ({ background, onChangeBackground }) => (
    <div className="ColorButts">
      <ul>
        change background :
        {backgroundColors.map((bg, index) => {
          const colorButtsStyle = {
            background: bg.color,
            color: Color(bg.color).isDark() ? "#FFF" : "#000"
          };
          return (
            <li
              key={index}
              onClick={()=>onChangeBackground(bg.color)}
              className={background === bg.color ? "selected" : ""}
              style={colorButtsStyle}
            >
              {bg.color}
            </li>
          );
        })}
      </ul>
      <div className="ColorPickerContainer">
        <HuePicker color={background} onChange={(color)=>onChangeBackground(color.hex)} />
      </div>
    </div>
  );

```

Reply:
@jonny
already implemented on #23